internal/command/outputs.go

```diff
@@ -1,7 +1,15 @@
 package command
 
-// Output stores command output.
-type Output struct {
-	Stdout string
-	Stderr string
+// Outputs is a collection of Output structs.
+type Outputs map[string]Output
+
+// Stdout returns the output Stdout by the Output ID.
+func (o Outputs) Stdout() map[string]string {
```

Comment:
```suggestion
func (o Outputs) Stdout() (out map[string]string) {
```

Don't apply this suggestion directly but this is the most concise way to initialize an empty return var. Alternatively when declaring use `make` to size the new map up front. Former is simplest latter is most performant. 

Not that it really matters here, just FYI

Reply:
Ah still getting a feel for when to use that. Figured it was nicer to do it in one line instead of two? You'd go with initialize in the signature and declaring at the top of the func instead?